docs/building.md

```diff
@@ -8,68 +8,69 @@ drivers.
 
 ### Windows
 
-* Windows 7 or later
-* [Visual Studio 2022, Visual Studio 2019, or Visual Studio 2017](https://www.visualstudio.com/downloads/)
+* [Visual Studio 2022,](https://www.visualstudio.com/downloads/) [Visual Studio 2019, or Visual Studio 2017](https://visualstudio.microsoft.com/vs/older-downloads/)
+  * You must choose `Desktop development with C++` in the `Workloads` tab of the Visual Studio installer.
+  * Windows 11 SDK version 10.0.22000.0 (this or any newer version)
   * For Visual Studio 2022, MSBuild `v142` must be used due to a compiler bug; See [#2003](https://github.com/xenia-project/xenia/issues/2003).
 * [Python 3.6+](https://www.python.org/downloads/)
   * Ensure Python is in PATH.
-* Windows 11 SDK version 10.0.22000.0 (for Visual Studio 2019, this or any newer version)
-
+  * Also see https://learn.microsoft.com/en-us/windows/python/faqs#why-does-running-python-exe-open-the-microsoft-store-
 ```
 git clone https://github.com/xenia-project/xenia.git
 cd xenia
-xb setup
+.\xb setup
```

Comment:
Backslashes are Windows-specific, I'd recommend forward slashes, which are accepted by both Linux and Windows (also considering that they're even more important on Linux than on Windows, possibly only for PowerShell on the latter, as on Linux, not providing an explicit executable path results in the executable being searched for only in the system directories or something like that).

Reply:
Forward slashes break cmd/Command Prompt. Should I just leave them out instead?

Reply:
They work in the Command Prompt as normal, possibly with the exception of UNC paths, although I'm not sure if this case applies to relative paths at all.

Reply:
![](https://user-images.githubusercontent.com/3462541/213912527-0d9ac8d9-23ad-4b0f-a224-aa4e74c95a15.png)

Reply:
That's not how you use it